index.html

```diff
@@ -1,26 +1,36 @@
-<DOCTYPE html>
+<!DOCTYPE html>
 <html>
     <head>
-        <meta charset="utf-8">
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
         <title>Calculator</title>
         <link rel="stylesheet" href="css/style.css" type="text/css">
-        <script src="./js/script.js"></script>
-    </head>
+    </head> 
     <body>
         <div class="calculator">
-            <form class="calculator-form">
-                <div class="calculator-form__input">
-                    <input type="text" id="a" value="1">
-                    <input type="text" id="b" value="1"> 
-                </div>
-                <div class="calculator-form__action-bar action-bar"> 
-                   <input type="button" class="action-bar__button action-bar__button-sum" id="calc-sam" onclick="calcSum();" value="Sum">
-                   <input type="button" class="action-bar__button action-bar__button-sub" id="calc-sub" onclick="calcSub();" value="Sub">
-                   <input type="button" class="action-bar__button action-bar__button-mul" id="calc-mul" onclick="calcMul();" value="Mul">
-                   <input type="button" class="action-bar__button action-bar__button-div" id="calc-div" onclick="calcDiv();" value="Div">
-                </div>
-            </form>
-            <div class="calculator-result" id="result" ></div>
+            <div class="calculator-form action-bar">
```

Comment:
I'm not sure that `action-bar` is the appropriate class name for this `div`. Please, rethink class names for that form.

Reply:
You can use the following BEM-structure:

```
-> .calculator-container
----> .calculator
--------> .calculator__output
--------> .calculator__action-bar
------------> .calculator-action-bar__item
----> .calculator-footer
```